RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -152,17 +154,29 @@ class ManageSubscriptionsViewModel: ObservableObject {
             self.loadingPath = nil
         }
     }
+
+    func handleInAppBrowserDismiss() {
```

Comment:
+1, I'd go with `onDismissInAppBrowser` before `dismissInAppBrowser` if we're handling something that already happened rather than doing it ourselves in the method, though